src/style.css

```diff
@@ -0,0 +1,29 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
```

Comment:
don't forget about the last star in each row 😉

<img width="137" alt="image" src="https://user-images.githubusercontent.com/55802630/173011093-4752d9d1-860d-403e-a191-7c3c052b19b7.png">